tests/resources/app_with_good_numeric_badge/suite.xml

```diff
@@ -0,0 +1,180 @@
+
+  <detail id="m1_case_short">
+    <title>
+      <text>With focus function</text>
+    </title>
+    <focus function="case_name = 'Fake Case Name'"/>
+    <field>
+      <header>
+        <text>Name</text>
+      </header>
+      <template>
+        <text>
+          <xpath function="case_name"/>
+        </text>
+      </template>
+      <sort type="string" order="1" direction="ascending">
+        <text>
+          <xpath function="case_name"/>
+        </text>
+      </sort>
+    </field>
+  </detail>
+
+  <entry>
+    <form>http://commcarehq.org/test/placeholder</form>
+    <command id="m0-f0">
+      <text>Form</text>
+    </command>
+    <instance id="casedb" src="jr://instance/casedb"/>
+    <session>
+      <datum id="case_id" nodeset="instance('casedb')/casedb/case[@case_type='test_case'][@status='open']" value="./@case_id" detail-select="m0_case_short" detail-confirm="m0_case_long"/>
+    </session>
+  </entry>
+
+  <entry>
+    <form>http://commcarehq.org/test/placeholder</form>
+    <command id="m0-f1">
+      <text>Form</text>
+    </command>
+    <instance id="casedb" src="jr://instance/casedb"/>
+    <session>
+      <datum id="case_id" nodeset="instance('casedb')/casedb/case[@case_type='test_case'][@status='open']" value="./@case_id" detail-select="m0_case_short" detail-confirm="m0_case_long"/>
+    </session>
+  </entry>
+
+  <remote-request>
+    <post url="http://fake.com/claim_patient/">
+      <data key="selected_name" ref="instance('patients')/case/name"/>
+      <data key="selected_case_id" ref="instance('patients')/case/@case_id"/>
+    </post>
+    <instance id="casedb" src="jr://instance/casedb"/>
+    <command id="patient-search">
+      <display>
+        <text>Global search for person</text>
+      </display>
+    </command>
+    <session>
+      <query url="http://fake.com/patient_search/" storage-instance="patients">
+        <data key="device_id" ref="instance('session')/session/data/uuid"/>
+        <data key="device_case_count" ref="instance('session')/session/data/case_count"/>
+        <prompt key="name">
+          <display>
+            <text>Input patient name</text>
+          </display>
+        </prompt>
+        <prompt key="patient_id">
+          <display>
+            <text>Input patient id</text>
+          </display>
+        </prompt>
+      </query>
+      <datum id="case_id" nodeset="instance('patients')/case[@case_type='geriatric'][@status='open']" value="./@case_id" detail-select="m0_case_short" detail-confirm="m0_case_long"/>
+    </session>
+    <stack>
+      <push>
+        <command value="'m1-f0'"/>
+        <datum id="calculated_data" value="'claimed'"/>
+      </push>
+    </stack>
+  </remote-request>
+
+  <menu id="m0">
+    <text>Menu</text>
+    <command id="m0-f0"/>
+  </menu>
+
+  <menu id="root" style="grid">
+    <text/>
+  </menu>
+
+  <menu id="m1" style="list">
+    <display>
+      <text>Menu 1 Text</text>
+      <text form="badge">
+        <xpath function="8"/>
```

Comment:
To Fix: this won't actually test whether the evaluation context for the badge is set correctly. I'd make sure this evaluating against session or case data or something similar.